cli/purge.go

```diff
@@ -0,0 +1,38 @@
+package cli
+
+import (
+	"github.com/stelligent/mu/common"
+	"github.com/stelligent/mu/workflows"
+	"github.com/urfave/cli"
+	"os"
+)
+
+func newPurgeCommand(ctx *common.Context) *cli.Command {
+	cmd := &cli.Command{
+		Name:    "purge",
+		Aliases: []string{"nuke"},
+		Usage:   "purge",
+		Flags: []cli.Flag{
+			cli.BoolFlag{
+				Name:  "yes, y",
+				Usage: "suppresses (Y/N) confirmation prompt",
+			},
+			cli.StringFlag{
+				Name:  "namespace, n",
+				Usage: "specify a namespace to filter",
+			},
+		},
+		Action: func(c *cli.Context) error {
+			paramName := "yes"
+			suppressConfirmation := c.Bool(paramName)
+			if suppressConfirmation {
+				ctx.ParamManager.SetParam("suppressConfirmation", "yes")
```

Comment:
ParamManager isn't the right way to pass args into a workflow.  This will end up writing a parameter into SSM parameter store named 'suppressconfirmation'.  Likely not what you intended.

Reply:
Thanks for the heads-up. I didn't realize it would modify AWS resources. I'll find another way.

Reply:
fixed in db4d0b. No more ParamManager for this parameter.